regulations/views/partial.py

```diff
@@ -69,6 +69,7 @@ def transform_context(self, context, builder):
         context['tree'] = {'children': [child_of_root]}
         context['navigation'] = self.section_navigation(
             context['label_id'], context['version'])
+        context['page_type'] = 'reg-section'
```

Comment:
Should this be part of the `navigation` field? Seems like it'll be easy to get out of sync otherwise.

Related: mind adding this to `views.diff.PartialSectionView` and `views.partial_interp.PartialSubterpView` for consistency?